src/iostream.c

```diff
@@ -174,23 +179,23 @@ static UInt OpenPty(int * master, int * slave)
        */
     *master = posix_openpt(O_RDWR | O_NOCTTY);
     if (*master < 0) {
-        Pr("OpenPty: posix_openpt failed\n", 0L, 0L);
```

Comment:
Can't Pr be redirected, while perror wouldn't be?

Reply:
Maybe. But isn't that a good thing?

Reply:
Well normally we assume tests can check errors, if any of these were printed, the tests would still "pass" (although probably fail for some other reason). I find it a bit weird we print these errors, but then continue., which isn't like (as far as I know?) any other error.

Reply:
I've addressed this, the errors are now printed through `Pr` again.